tests/src/test/scala/common/Wsk.scala

```diff
@@ -1070,6 +1060,16 @@ trait RunWskCmd extends BaseRunWsk {
 
     rr
   }
+
+  /** Retries cmd on network error exit. */
+  private def retry(i: Int, N: Int, cmd: () => RunResult): RunResult = {
+    val rr = cmd()
+    if (rr.exitCode == NETWORK_ERROR_EXIT && i < N) {
+      Thread.sleep(1.second.toMillis)
+      println(s"command will retry to due to network error: $rr")
```

Comment:
Shall we use log this message as a warning message or info? I do not see println is still using anywhere else except in the test cases.